package.json

```diff
@@ -49,5 +49,7 @@
     "rimraf": "^2.6.3"
   },
   "license": "MIT",
-  "dependencies": {}
+  "devDependencies": {
+    "@types/node": "^12.0.4"
```

Comment:
```suggestion
    "@types/node": "^12.0.4",
    "chai": "^4.2.0",
    "eslint": "^5.16.0",
    "mkdirp": "^0.5.1",
    "mocha": "^6.1.4",
    "npm-run-all": "^4.1.5",
    "pegjs": "^0.10.0",
    "rimraf": "^2.6.3"
```

Reply:
oops! fixed.

Reply:
`@types/node` should be the first entry in the `devDependencies` record to ensure that it doesn’t get re‑sorted when `npm i ‑D <foo>` gets run, which would lose `git blame` information.